index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <section id="welcome-section">
+        <h1>Lorem Ipsum</h1>
+    </section>
+    <nav id="navbar" ><a href="#welcome_section"></a></nav>
```

Comment:
# Good job showing your understanding of semantic html tags!
- You used the semantic 'nav' tag to show your navigation bar ✔️ 

## You should however add objects and a menu to enable navigation from your navbar.